azure-devops/azext_devops/test/common/test_pip_helper.py

```diff
@@ -12,7 +13,26 @@ def test_run_pip_is_present_in_cli_core(self):
             from azure.cli.core.extension.operations import _run_pip
         except ImportError:
             self.fail('dependency on cli core is broken, pip helper will need fix')
+            
+    def test_keyring_install_working(self):
+        install_keyring()
+        # uninstall keyring
+        pip_args = ['uninstall', 'keyring', '-y']
+        from azure.cli.core.extension.operations import _run_pip
+        pip_status_code = _run_pip(pip_args)  # pylint: disable=protected-access
+        if pip_status_code > 0:
+            self.fail('keyring uninstall failed')
 
+        try:
+            from imp import reload
```

Comment:
https://stackoverflow.com/questions/961162/reloading-module-giving-nameerror-name-reload-is-not-defined

This suggested that reload is not in the same import for different python versions, but the tests passed for this PR in different env. Please check it.

The test is passing due to different exceptions, do a selecting catch not generic.